BitmovinAnalyticsCollector/Classes/AVPlayer/Classes/AVPlayerAdapter.swift

```diff
@@ -79,20 +78,21 @@ class AVPlayerAdapter: NSObject, PlayerAdapter {
                 stateMachine.transitionState(destinationState: .playing, time: player.currentTime())
             }
 
-            break
         case .failed:
             errorOccured(error: playerItem.error as NSError?)
-            break
+
         default:
             break
         }
     }
-    
+
     private func errorOccured(error: NSError?) {
         let errorCode = error?.code ?? 1
         let errorMessage = error?.localizedDescription ?? "Unkown"
-        
-        stateMachine.transitionState(destinationState: .error, time: player.currentTime(), data: [BitmovinAnalyticsInternal.ErrorCodeKey: errorCode, BitmovinAnalyticsInternal.ErrorMessageKey: errorMessage])
+
+        stateMachine.transitionState(destinationState: .error,
+                                     time: player.currentTime(), data: [BitmovinAnalyticsInternal.ErrorCodeKey: errorCode,
```

Comment:
`time:` parameter from the new line but not he `data:`?

Reply:
will be mich better readable, yes. I just shortened the line length.